### PR TITLE
Permissions: Cookies correspond to domain, difference between the "default value" and an "actual value" (improvements)

### PR DESCRIPTION
**Pull request by janekptacijarabaci**
Ad https://github.com/MoonchildProductions/Pale-Moon/issues/823#issuecomment-294313936

> 1) Cookies correspond to strict urls, not to domains as on single page permissions
> 2) There are no easy way to find which permissions were changed on which pages

@JustOff
If that's how you meant (partially)...

While it is true that:
> There are no easy way to find which permissions were changed on which pages

...difference between the "default value" and an "actual value" - it may be displayed.

---

I've created the new build (x32, Windows) and tested.

### CODE REVIEW

File: browser/themes/osx/preferences/aboutPermissions.css [hunk 1]
```diff
@@ -118,6 +118,13 @@
   font-size: 125%;
   margin-bottom: 0;
   font-weight: bold;
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.pref-default {
+  margin-left: 1ex;
```
Comment (wolfbeast) [inline browser/themes/osx/preferences/aboutPermissions.css]: You should probably use `em` here, not `ex`...

File: browser/themes/linux/preferences/aboutPermissions.css [hunk 2]
```diff
@@ -130,4 +137,13 @@
 .plugins-vulnerable {
   margin-left: 0;
   padding-left: 0;
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.plugins-default {
+  margin-left: 1ex;
```
Comment (wolfbeast) [inline browser/themes/linux/preferences/aboutPermissions.css]: You should probably use em here, not ex...

File: browser/themes/linux/preferences/aboutPermissions.css [hunk 3]
```diff
@@ -115,6 +115,13 @@
   font-size: 125%;
   margin-bottom: 0;
   font-weight: bold;
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.pref-default {
+  margin-left: 1ex;
```
Comment (wolfbeast) [inline browser/themes/linux/preferences/aboutPermissions.css]: You should probably use em here, not ex...

File: browser/themes/osx/preferences/aboutPermissions.css [hunk 4]
```diff
@@ -134,4 +141,13 @@
 .plugins-vulnerable {
   margin-left: 0;
   padding-left: 0;
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.plugins-default {
+  margin-left: 1ex;
```
Comment (wolfbeast) [inline browser/themes/osx/preferences/aboutPermissions.css]: ditto

File: browser/themes/windows/preferences/aboutPermissions.css [hunk 5]
```diff
@@ -118,6 +118,13 @@
   font-size: 125%;
   margin-bottom: 0;
   font-weight: bold;
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.pref-default {
+  margin-left: 1ex;
```
Comment (wolfbeast) [inline browser/themes/windows/preferences/aboutPermissions.css]: ditto

File: browser/themes/windows/preferences/aboutPermissions.css [hunk 6]
```diff
@@ -134,4 +141,13 @@
 .plugins-vulnerable {
   margin-left: 0;
   padding-left: 0;
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.plugins-default {
+  margin-left: 1ex;
```
Comment (wolfbeast) [inline browser/themes/windows/preferences/aboutPermissions.css]: ditto

**Comment by wolfbeast** [comment]
I just built this and it's not good. Disabling the options for plugins that are set to default is NOT good because people will not be able to change the permissions, which is the whole point of having the permissions manager in place... What were you thinking?

**Comment by janekptacijarabaci** [comment]
OK. You can change it according to your needs :-)
https://github.com/janekptacijarabaci/Pale-Moon/blob/248229018be44049493f2e3ecb16959704afaa20/browser/components/preferences/aboutPermissions.js#L549

**Comment by wolfbeast** [comment]
My mistake -- I didn't really understand what you were doing and misinterpreted the output.

**Pull request merged**